cmd/server/assets/admin/events/index.html

```diff
@@ -33,7 +33,8 @@ <h1>System event log</h1>
       </div>
 
       <div class="card-body">
-        <form method="GET" action="/admin/events" id="search-form">
+        <form method="GET" id="search-form">
+          <!-- There's a hidden query param to filter on test `include_test=true` -->
```

Comment:
Note: golang html template doesn't render comments.

That may or may not be desirable. Then again we also have golang `{{/* comment */}}` comments, so that seems like a weird choice. Ask me how I feel about golang templating sometime.